nixos/modules/services/misc/numworks.nix

```diff
@@ -0,0 +1,28 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.numworks;
+in {
+
+  ###### interface
+
+  options = {
```

Comment:
Seems adding a module for this is overkill. There are a lot of hardware-related packages that come with `udev` rules, where we ask people to just put it in `services.udev.packages`.

Reply:
Yep, I've removed the module.